deployment/README.md

```diff
@@ -0,0 +1,4 @@
+# General Deployment files
+Each apllication has its own folder with Dockerfile and docker_compose file. The makefile is used to push image to ecr and create a new task definition. To get the latest service running, manually update the service in the console using the task definition with revision number that was output after creating the task definition.
+The above process will be automated in Github Actions.
+IF the docker_compose does not change, the task definition revision number will remain the same. To run ecs-cli, you will need to add the aws profile to (~/.aws/credentials) since it does not work with SSO
```

Comment:
Where do I get this token?

Reply:
Note to self: need to install `ecs-cli` to be able to run this
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_CLI.html